### DISCUSSION TITLE
the restart will trigger the countdown

### DISCUSSION THREAD

**Discussion by sommo** [micbay/RaceTimerAndController#13] [General]
Hi @micbay,
it could be nice/possible to set the second press of the pausebutton to restart the race but with the countdown like the one on the start? cause in that way, after pressed the button you can sit back again and get ready to race :)

Thank you!

**Comment by micbay** [comment]
This feature has been added in [version 2.0+](https://github.com/micbay/RaceTimerAndController/releases/tag/v2.0.0).
The default configuration is still to have no countdown, however you can set `CTDWN_ON_RESTART` to `true` in your `localSettings.h` file to enable a restart countdown.
Let me know if this does not satisfy the above request.

**Reply by sommo** [reply]
wow perfect, thank you! in pause mode could also be funny to see the ledbar blynking orange like "something went wrong on the track and man are at work to get car back on track" :D anyway is a minor suggest
